src/app/loginComponents/requests/organization-delete-confirm-dialog.html

```diff
@@ -0,0 +1,10 @@
+<h1 mat-dialog-title>Delete Organization</h1>
```

Comment:
Why can't you use the existing confirmation dialog component?  If it's missing something, and not too much work, I think it would be better to update that component rather than create a special one-off version.

Reply:
Done. Modified the existing confirmation dialog component and is now implemented with the organization deletion function.

Reply:
i was here, this doesn't look addressed